R/min_grid.R

```diff
@@ -46,13 +46,6 @@
 #'   grid_regular(levels = 3)
 #'
 #' min_grid(xgb_spec, xgb_grid)
-#' @export
```

Comment:
It might be good to have an explicit `NULL` after the last `#'` line.

I think this signals to roxygen that you are done with this free floating chunk of documentation

You'll probably need to add `@name min_grid` if you do that. I have a feeling that this chunk of documentation is matching up to the `min_grid.model_spec()` code below

Reply:
Like this
https://github.com/DavisVaughan/almanac/blob/dc1e4d788fc8ab6557b95b76035437782e1841bd/R/adjustments.R#L49-L80

Reply:
I think we _have_ to have the documentation match up to one of the methods. Otherwise, if we add a `@name`, it gives an error like `Rd files with duplicated alias 'min_grid'`. I think the reason it works for your almanac example is that you are not dealing with something reexported. Maybe a better thing to look at is [fit](https://github.com/tidymodels/parsnip/blob/master/R/fit.R) or [tidy](https://github.com/tidymodels/recipes/blob/master/R/tidy.R).

Reply:
Okay thats fine. Can you move the block of pure comments starting with `As an example` up above the documentation? That way you can link the min-grid docs to the `min_grid.model_spec` method more clearly